iotqatools/cb_v2_utils.py

```diff
@@ -348,7 +346,10 @@ def __create_attributes(self, entity_context):
 
     def create_entities(self, context, entities_number, attributes_number):
```

Comment:
The context is a dict with the features of the request. See https://github.com/telefonicaid/iot-qa-tools/blob/develop/iotqatools/cb_v2_utils.py#L72